src/api/app/views/webui2/webui/project/_side_links.html.haml

```diff
@@ -33,8 +33,8 @@
 
   - if linking_projects.present?
     %li
-      %i.fa.fa-info-circle.text-info
-      = link_to(linking_projects_path(project), remote: true) do
+      = link_to('#', class: 'nav-link', data: { toggle: 'modal', target: '#linking-projects-modal' }) do
```

Comment:
`nav-link` breaks the alignment

![remove-nav-link](https://user-images.githubusercontent.com/1102934/53556347-e9a0fc00-3b43-11e9-9a3b-006717f814b0.png)